code/game/objects/structures/barricade/barricade.dm

```diff
@@ -252,7 +254,7 @@
 	if(P.ammo.damage_type == BURN)
```

Comment:
Please also fix the single letter vars here. See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#variables
You can use VSC to do this easily by selecting the function, search, enable dropdown for replace, enable word, enable case, and then just for P and replace with `bullet`or something.

Reply:
Done...

Reply:
Or not